nixos/modules/services/monitoring/glpi-agent.nix

```diff
@@ -0,0 +1,69 @@
+{
+  config,
+  lib,
+  pkgs,
+  ...
+}:
+
+let
+  cfg = config.services.glpiAgent;
+
+  format = pkgs.formats.keyValue {
+    listsAsDuplicateKeys = true;
```

Comment:
This doesn't seem correct? According to the documentation, and also your own special casing of the `servers` option, multiple values should be comma separated. Fixing this would also allow removing the explicit `servers` option.

Reply:
Fixed now not have dedicated server options.

Reply:
Why are you special casing it at all? I don't see any indication that the server entry has to go first.